app/public/js/home.js

```diff
@@ -64,6 +64,7 @@ function initMap()
 	});
 	google.maps.event.addListener(map, 'mousedown', function(event) {
 		locationInfo(map, event.latLng);
+		blinkMap(document.getElementById('map'))
```

Comment:
When I switch to that I get this error
![image](https://user-images.githubusercontent.com/19508538/42336744-9a9e5b54-8052-11e8-8ea1-da873c16fec6.png)
I am not able to access the style property

Reply:
Wups! I messed up my suggestion. I suggested $('map')[0] originally when it should have been $('map')[0].

The selector follows CSS-selector rules so not including the '#' selected by tag name instead of id.

Try $('#map')[0] now.

Reply:
All good now, thanks for the help